src/cache_cf.cc

```diff
@@ -2515,7 +2511,7 @@ parse_peer_access(void)
         return;
     }
 
-    CachePeer *p = peerFindByName(host);
+    const auto p = findCachePeerByName(host);
```

Comment:
Done in 52d451b.

FWIW, I do not see how upgrading fatal() to throw implies that this PR is the right place to reject configurations with incorrect cache_peer references, but I certainly agree that we should reject such configurations. Now we do.

Reply:
Please note that peername and peername_regex ACLs still ignore peer-name typos. I think they should be fixed as well, but perhaps that is best done in a dedicated PR? It will require quite a few out-of-scope changes, especially if we want to at least warn the admin if peername_regex cannot match _any_ of the configured cache_peers.

Reply:
Up to you, I took the debugs() as being in-scope due to the changes with detecting and reporting other details on cache-peer and cache_peer_access specifically. If you want to call the ACL value validation out of scope, thats fine.